Recrutify/Recrutify.Services/Validators/CandidatesIdValidator.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentValidation;
+using Recrutify.DataAccess.Models;
+using Recrutify.DataAccess.Repositories.Abstract;
+using Recrutify.Services.DTOs;
+
+namespace Recrutify.Services.Validators
+{
+    public class CandidatesIdValidator : AbstractValidator<CreateBullFeedbackTestDTO>
+    {
+        public CandidatesIdValidator(ICandidateRepository candidateRepository)
+        {
+            RuleFor(x => x)
```

Comment:
RuleForEach for each candidate id, use MustAsync, and cause you have complicated logic better to put not anonymous method in MustAsync, but define separate method